### PR TITLE
blackbox: Add accumulating feedback types for numeric and text values.

### PR DESCRIPTION
**Pull request by warriorstar-orion**
## What Does This PR Do
This PR adds two new feedback types to blackbox, `ledger` and `nested ledger`. These can be used when needed to record discrete values of a given blackbox row. See the documentation added to `.github/USING_FEEDBACK_DATA.md` for the specifics.

## Why It's Good For The Game
There are rare cases when this level of granularity is needed, and it's better to pull it out into its own feedback type rather than try and shoehorn these cases into the existing feedback types.

## Testing

All JSON in the documentation was pulled directly from the test entries after running them through the database.

<hr>

### Declaration

- [X] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog
NPFC

### CODE REVIEW

File: SQL/paradise_schema.sql [hunk 1]
```diff
@@ -250,7 +250,7 @@ CREATE TABLE `feedback` (
   `datetime` datetime NOT NULL,
   `round_id` int(8) NOT NULL,
   `key_name` varchar(32) NOT NULL,
-  `key_type` enum('text', 'amount', 'tally', 'nested tally', 'associative') NOT NULL,
+  `key_type` enum('text', 'amount', 'tally', 'nested tally', 'associative', 'ledger', 'nested ledger') NOT NULL,
```
Comment (Burzah) [inline SQL/paradise_schema.sql:253]:
```suggestion
  `key_type` ENUM('text', 'amount', 'tally', 'nested tally', 'associative', 'ledger', 'nested ledger') NOT NULL,
```
Nit. (Slowly trying to change these, but not a big deal)

File: SQL/updates/59-60.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+ALTER TABLE
+	`feedback`
+MODIFY
+	COLUMN `key_type` enum(
```
Comment (Burzah) [inline SQL/updates/59-60.sql:4]:
```suggestion
	COLUMN `key_type` ENUM(
```
Nit.

File: SQL/updates/59-60.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+ALTER TABLE
```
Comment (Burzah) [inline SQL/updates/59-60.sql:1]: Can you include the update numbers (from 59 to 60), a brief description of the script, and your handle here at the top?

**Review by Burzah — COMMENTED** [review]
SQL stuff looks good to me, just had 2 nits and one request.

File: SQL/updates/59-60.sql (hunk 3 above)
Comment (warriorstar-orion) [reply SQL/updates/59-60.sql:10]: Done

File: SQL/updates/59-60.sql (hunk 2 above)
Comment (warriorstar-orion) [reply SQL/updates/59-60.sql]: Done

File: SQL/paradise_schema.sql (hunk 1 above)
Comment (warriorstar-orion) [reply SQL/paradise_schema.sql]: Done

**Review by Burzah — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Comment by AffectedArc07** [comment]
I'll handle the SQL changes when I am awake enough to

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**